src/generated/resources/data/forge/tags/worldgen/biome/is_cold.json

```diff
@@ -0,0 +1,21 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:taiga",
+    "minecraft:the_end",
```

Comment:
same deal here as the hot tag. Should all the other end biomes be added to this tag or split off into a cold_otherwordly biome tag? Make it easier for mods and datapacks using the mojang json structures system to add to cold overworld biomes.